tasks/vendoring/__init__.py

```diff
@@ -141,6 +159,106 @@ def vendor(ctx, vendor_dir):
         apply_patch(ctx, patch)
 
 
+def download_licenses(ctx, vendor_dir):
+    log('Downloading licenses')
+    tmp_dir = vendor_dir / '__tmp__'
+    ctx.run(
+        'pip download -r {0}/vendor.txt --no-binary :all: -d {1}'.format(
```

Comment:
You could add `--no-deps`.

Reply:
Does it download deps without that?

Reply:
No, but we don't vendor dependencies either (cf https://github.com/pypa/pip/blob/b6bbabe3e7bc633f7af9da35f55d8c9d0d7a3f09/tasks/vendoring/__init__.py#L105)

Reply:
Well I (wrongly) assumed `install` would bring dependencies, but `download` would not. Will add `--no-deps`.